### ISSUE TITLE
Needs to support addrs with /tcp/port too

### ISSUE THREAD

**Issue by diasdavid** [libp2p/js-libp2p-webrtc-star#129]
Similar issue to https://github.com/libp2p/js-libp2p-websocket-star/issues/35, also errors with:

```
Error: invalid multiaddr/dns4/wrtc-star.discovery.libp2p.io/tcp/443/wss/p2p-webrtc-star/ipfs/QmTysQQiTGMdfRsDQp516oZ9bR3FiSCDnicUnqny2q1d79
```

**Comment by diasdavid** [comment]
js-mafmt works well with that multiaddr -- https://github.com/whyrusleeping/js-mafmt/pull/20 --, the issue is on the module multiaddr parsing function

**Comment by diasdavid** [comment]
Solved with #130
